### DISCUSSION TITLE
Add List of Contributors to README

### DISCUSSION THREAD

**Discussion by DhanushNehru** [avelino/awesome-go#5086] [Ideas]
**Description:**
The README file is a vital component of any open-source project, serving as a primary source of information for users and potential contributors. To improve the transparency and recognition of those who have contributed to the project, we should add a list of contributors to the README.

This list of contributors can include their GitHub usernames or real names, along with a brief description of their contributions to the project. This not only acknowledges the hard work and dedication of our contributors but also encourages more people to get involved and contribute to the project.

By adding a list of contributors to the README, we aim to create a more inclusive and collaborative environment, foster a sense of community, and give credit where it's due. This simple yet effective addition can be achieved by updating the README file with the relevant information.

**Comment by phanirithvij** [comment]
This project has thousands of one-time contributors and as of now, browsers are struggling to render the readme. I don't think this is necessary.
